doc/spec/PluginAPI.md

```diff
@@ -0,0 +1,16 @@
+# Sia-UI Plugin API Specification
+
+## Introduction
+
+This specification outlines the functionality that Sia-UI's plugin API exposes to developers.
+
+## Desired Functionality
+
+The plugin API should provide an importable javascript module that provides all of the following features:
+
+- A function which registers a plugin with Sia-UI.
+- Access to the running, configured, `sia.js` wrapper.
+- Functions to open and save files on the host.
```

Comment:
should clarify this language, since "host" in this context does not mean a Sia host